Library/Formula/pmd.rb

```diff
@@ -25,6 +25,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{ENV["HOME"]}”
```

Comment:
No matter @tdsmith . I'll add it back in the `install` function.

And don't worry about sending me back and forth! GitHub literally ignored a pull request I sent for weeks until it became out-of-date, at which point I just closed it. A little back-and-forth is far better than that.